spec/lucky/action_route_params_spec.cr

```diff
@@ -17,3 +23,11 @@ describe "Automatically generated param helpers" do
     typeof(action.param_2).should eq String
   end
 end
+
+describe "Testing params that end with file extension" do
+  it "generates helpers for the param name" do
+    action = TestWithFileExtensionAction.new(build_context, {"param_1" => "param_1_value"})
+    action.param_1.should eq "param_1_value"
+    typeof(action.param_1).should eq String
```

Comment:
Could you test that the output is the correct string and not just the right type? Just to make sure it works correctly. Might be worth doing in the one above too just to be sure